blackbox/docs/general/builtins/scalar.rst

```diff
@@ -333,6 +333,26 @@ Examples
    +---------------------+
    SELECT 1 row in set (... sec)
 
+.. _scalar-quote-ident:
+
+``quote_ident(text)``
+---------------------
+
+Returns: ``text``
+
+Quotes a provided string argument. The string is quoted only when in is need.
```

Comment:
Should we stress the `ident` part in the description? eg mention that the intent of this function is that it can be used as an identifier in an `SQL` statement string. May add something similar to the release notes?